processor/runtime/src/test/java/com/redhat/service/smartevents/processor/actions/kafkatopic/KafkaTopicActionInvokerBuilderTest.java

```diff
@@ -11,22 +11,16 @@
 import org.apache.kafka.clients.admin.ListTopicsResult;
 import org.apache.kafka.common.KafkaFuture;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
```

Comment:
Should this class even has the suffix "Test" if there are no tests?

Reply:
It included a few tests before, I'll remove the class entirely